hydromt/models/model.py

```diff
@@ -313,127 +300,58 @@ def update(
             try to write to a file that's already opened. The output will be written
             to a temporary file in case the original file cannot be written to.
         """
-        opt = opt or {}
-        opt = self._check_get_opt(opt)
+        steps = steps or {}
+        validate_steps(self, steps)
+
+        # check if region.create is in the steps, and remove it.
+        if steps.pop("region.create", None) is not None:
+            self.logger.warning(
+                "region.create can only be called when building a model."
+            )
```

Comment:
keep the _CLI_ARGS method behavior

Reply:
see https://github.com/Deltares/hydromt/pull/845#discussion_r1535245634